scripts/zones/Bibiki_Bay/mobs/Dalham.lua

```diff
@@ -7,13 +7,44 @@ local entity = {}
 
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180)
+end
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:setMod(xi.mod.SLEEPRES, 90)
```

Comment:
This modifier is exclusively used for job traits and similar, which make it so it returns an special resist message, allowing us to know it was the job trait that triggered

There should be a MEVA mod for sleep

Reply:
> This modifier is exclusively used for job traits and similar, which make it so it returns an special resist message, allowing us to know it was the job trait that triggered
> 
> There should be a MEVA mod for sleep

Thanks, I have updated the PR!